crud.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// Inserting the record
+func (apiCfg apiConifg) insertRecord(record any) (interface{}, error) {
+	// Maybe extract the dbName and the collectionName from .env file?
+	collection := apiCfg.dbClient.Database("munch").Collection("Testing")
```

Comment:
as mention by @raamsri 

we have to wrap this code and create a function which does these.

Reply:
Hmm I'm not fully sure what u mean by wrapping the code..